rosco-core/rosco-core.gradle

```diff
@@ -11,4 +11,5 @@ dependencies {
   compile spinnaker.dependency('korkWeb')
   compile spinnaker.dependency('korkArtifacts')
   compileOnly spinnaker.dependency("lombok")
+  compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.3.0'
```

Comment:
Jackson should already be included in the `spinnaker.group('jackson')` dependency.

Reply:
This dependency is required, I didn't find the **jackson-dataformat-yaml**, where should be looking for?

Reply:
Is it possible to just use SnakeYAML in this case? I know this is going to get called out when we submit this upstream so I'd like to explore any options that don't require a new dependency.

Reply:
using SnakeYAML, fixed.